### ISSUE TITLE
Accessibility violation on CvFileUploader component

### ISSUE THREAD

**Issue by caroluchoa** [carbon-design-system/carbon-components-vue#1355]
## Detailed description

> Describe in detail the issue you're having.
This issue is related to CvFileUploader. Using IBM Accessibility Checker on the CvFileUploader results on the following violation: **"The tabbable element's role 'none' is not a widget role"**
I have uploaded a print of accessibility checker scan showing the violation and the code.

> Is this a feature request (new component, new icon), a bug, or a general issue?
Bug

> Is this issue related to a specific component?
CvFileUploader

> What did you expect to happen? What happened instead? What would you like to see changed?
Was expecting no violations. Got an accessibility violation as showed on the screenshot below.

> What browser are you working in?
Chrome

> What version of the Carbon Design System are you using?
"@carbon/vue": "^2.44.0"

## Additional information

- Screenshots or code
![image](https://user-images.githubusercontent.com/43659990/184219399-250ff7e0-d959-4fd1-9eb4-ff2d1275808a.png)

```html
  <cv-file-uploader
       ref="fileUploader"
       v-model="receiversFile"
       data-cy="fileUploader"
       class="email-drag-drop"
       kind="drag-target"
       accept=".xlsx"
       :clear-on-reselect="true"
       :removable="true">
 </cv-file-uploader>
```

- Notes
